libs/utils/analysis/tasks_analysis.py

```diff
@@ -636,7 +671,7 @@ def _plotTaskSignals(self, axes, tid, signals, is_last=False):
         :type is_last: bool
         """
         # Get dataframe for the required task
-        util_df = self._dfg_trace_event('sched_load_avg_task')
+        util_df = self._dfg_task_lt_events()
```

Comment:
Still missing check for `util_df not None`?
Like the one you have at line 581

Reply:
This one isn't strictly needed as it's checked by the caller. But I'm gona add it in anyway, rather than have invisible dependencies between methods like that. Thanks for pointing it out.